index.html

```diff
@@ -4,25 +4,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Kalolo's Portfolio</title>
-    
     <link rel="stylesheet" href="styles.css">
-
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+    crossorigin="anonymous">
+    <title>Kalolo's Portfolio</title>
 </head>
 <body>
 <header>
     <nav class="nav-bar">
 
     <div class="logo">
-        <a href="#">My logo</a>
+        <a href="">My logo</a>
     </div>
         <div class="menu-icon"> <img src="images/menu.png" alt="menu navbar"></div>
     </nav>
 </header>
 
     <section class="first-section" id="section-one">
         <div class="text">
-            <h1>
+            <h1 class="salutation">
                 Hi I'm Kalolo, I'm glad to see you!
             </h1>
             <p class="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] Fix margin/padding in your first section according to the Figma design you selected.
- [ ] Fix margin between sections.
- [ ] Fix padding/margin for "Works" section.

Figma design | Yours

 ![image](https://user-images.githubusercontent.com/60164239/129461965-2cc020d9-e155-445d-901f-a33a3d3df7b5.png)